storage/storage.go

```diff
@@ -84,3 +84,41 @@ func (st *Storage) CheckFundedUserWithWaitTime(userID []byte, authType string) (
 	wp := binary.LittleEndian.Uint64(wpBytes)
 	return wp >= uint64(time.Now().Unix()), time.Unix(int64(wp), 0)
 }
+
+// AddPendingStripeSession adds a pending Stripe session to the storage.
+// It takes a sessionID string as a parameter and returns an error if any.
+func (st *Storage) AddPendingStripeSession(sessionID string) error {
+	tx := st.kv.WriteTx()
+	defer tx.Discard()
+	if err := tx.Set([]byte(sessionID), []byte("true")); err != nil {
+		log.Error(err)
```

Comment:
Don't log the errors, here. If the function returns an error, just return it. The caller will decide if to print it or not.
In addition we don't want to run tx.Commit() if tx.Set() fails.